source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,281 @@
+.. _swupd-3rd-party:
+
+Create, Install, and Manage 3rd-party Bundles
+#############################################
+
+Upstream Clear Linux offers a plethora of bundles to choose from.  
+As of version 32570, there are over 1000 bundles and counting.  
+
+To provide you access to even more software outside of the distro, we have 
+enabled two new features to our :command:`mixer` and :command:`swupd` client tools
```

Comment:
Make the first references of `mixer` and `swupd` `:ref` links to their documentation pages.

Reply:
done.